notes/src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
```

Comment:
Lets separate imports:
```suggestion
import React from "react";
import ReactDOM from "react-dom";

import App from "./App";
import "./index.css";
```